.github/workflows/eamxx-sa-testing.yml

```diff
@@ -12,6 +12,7 @@ on:
       - 'cime_config/machine/config_machines.xml'
       - 'components/eamxx/**'
       - 'components/homme/**'
+      - 'components/eam/src/physics/rrtmgp/**'
```

Comment:
Minor, but the submodule is in `components/eam/src/physics/rrtmgp/external`. The other stuff in the rrtmgp folder is unused by eamxx (that I can tell). Same for the other workflow file.